src/vmm/tests/integration_tests.rs

```diff
@@ -133,3 +135,88 @@ fn test_vmm_seccomp() {
         }
     }
 }
+
+#[test]
+fn test_dirty_bitmap_error() {
+    // Error case: dirty tracking disabled.
+    let pid = unsafe { libc::fork() };
+    match pid {
+        0 => {
+            set_panic_hook();
+            let boot_source_cfg: BootSourceConfig =
+                MockBootSourceConfig::new().with_default_boot_args().into();
+            let resources: VmResources = MockVmResources::new()
+                .with_boot_source(boot_source_cfg)
+                .into();
+            let mut event_manager = EventManager::new().unwrap();
+            let empty_seccomp_filter = get_seccomp_filter(SeccompLevel::None).unwrap();
+
+            let vmm = build_microvm(&resources, &mut event_manager, &empty_seccomp_filter).unwrap();
+            assert_eq!(
+                format!("{:?}", vmm.lock().unwrap().get_dirty_bitmap().err()),
+                "Some(DirtyBitmap(Error(2)))"
```

Comment:
nit: maybe add a comment explaining that if the requested bitmap doesn't exist, the kernel returns ENOENT (2). On a cursory glance that 2 looks a bit mysterious 😄